### ISSUE TITLE
JUnit plugin import throws error when trying to import multi testsuite node XML

### ISSUE THREAD

**Issue by nabeykoon** [reportportal/reportportal#2395]
Earlier we used the Service level - launch controller Import API to successfully import junit XML generated during Cypress test execution. (attached sample XML)
This XML was generated using 2 steps,
1. Generate junit XML for each spec file we execute. (using package: mocha-junit-reporter)
2. Merge all XML generated for each spec file together (using package: junit-report-merger)

But when I try to use the new Junit plugin along with new Report portal version 24.2, it gave me following error message while importing (which was successfully imported when using Service API)

{
  "errorCode": 40035,
  "message": "Error while importing the file. 'Start time of child ['2024-10-24T21:07:32Z'] item should be same or later than start time ['2024-10-24T21:07:37Z'] of the parent item/launch '1105''"
}

It seems the plugin validates testsuite timestamps as parent-child even though they are sibling nodes under parent node"testsuites"  

Could you please clarify if this validation is the intended response of Junit plugin? 
[api-grubcenter-smoke-test-execution.zip](https://github.com/user-attachments/files/17584661/api-grubcenter-smoke-test-execution.zip)

**Comment by Pink-Bumblebee** [comment]
@nabeykoon , currently we are working on fix for this bug.

**Comment by Pink-Bumblebee** [comment]
@nabeykoon , we added some fixes into api-service and junit import plugin. Please update your env and check again.
From my side I checked your file at demo.reportportal.io - everything has imported without errors.

**Comment by nabeykoon** [comment]
@Pink-Bumblebee, I tried to update api-service and spin up the environment. But I got the following error when starting container "reportportal/service-api:5.12.1" which is complaining about the same Junit plugin version already uploaded
Notes: 
Earlier while I'm troubleshooting the above error, I have manually uploaded Junit plugin to this environment. 
Also this environment is upgraded through multiple versions of Report portal. (ver 5.7.2 -> ver 24.1 -> ver 24.2)

For the time being, I will test this in new test environment and let you know the results for this specific fix.

=============================container log==========================================
2024-11-18 13:36:55 Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
2024-11-18 13:36:55 2024-11-18 08:06:55.232 ERROR 1 --- [           main] o.s.boot.SpringApplication               : Application run failed
2024-11-18 13:36:55 
2024-11-18 13:36:55 org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'pluginStartUpService': Invocation of init method failed; nested exception is com.epam.reportportal.rules.exception.ReportPortalException: Error during plugin uploading: 'Plugin with ID = 'junit' of the same VERSION = '1.0.0' has already been uploaded.'
2024-11-18 13:36:55     at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeInitialization(InitDestroyAnnotationBeanPostProcessor.java:160)
2024-11-18 13:36:55     at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyBeanPostProcessorsBeforeInitialization(AbstractAutowireCapableBeanFactory.java:440)
2024-11-18 13:36:55     at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1796)
2024-11-18 13:36:55     at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:620)
2024-11-18 13:36:55     at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542)
2024-11-18 13:36:55     at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335)
2024-11-18 13:36:55     at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
2024-11-18 13:36:55     at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333)
2024-11-18 13:36:55     at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208)
2024-11-18 13:36:55     at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:955)
2024-11-18 13:36:55     at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:920)
2024-11-18 13:36:55     at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:583)
2024-11-18 13:36:55     at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:145)
2024-11-18 13:36:55     at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:780)
2024-11-18 13:36:55     at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:453)
2024-11-18 13:36:55     at org.springframework.boot.SpringApplication.run(SpringApplication.java:343)
2024-11-18 13:36:55     at org.springframework.boot.SpringApplication.run(SpringApplication.java:1370)
2024-11-18 13:36:55     at org.springframework.boot.SpringApplication.run(SpringApplication.java:1359)
2024-11-18 13:36:55     at com.epam.ta.reportportal.core.configs.ReportPortalApp.main(ReportPortalApp.java:38)
2024-11-18 13:36:55     at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
2024-11-18 13:36:55     at java.base/java.lang.reflect.Method.invoke(Method.java:580)
2024-11-18 13:36:55     at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49)
2024-11-18 13:36:55     at org.springframework.boot.loader.Launcher.launch(Launcher.java:108)
2024-11-18 13:36:55     at org.springframework.boot.loader.Launcher.launch(Launcher.java:58)
2024-11-18 13:36:55     at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:88)
2024-11-18 13:36:55 Caused by: com.epam.reportportal.rules.exception.ReportPortalException: Error during plugin uploading: 'Plugin with ID = 'junit' of the same VERSION = '1.0.0' has already been uploaded.'
2024-11-18 13:36:55     at com.epam.reportportal.rules.commons.validation.ErrorTypeBasedRuleValidator.verify(ErrorTypeBasedRuleValidator.java:31)
2024-11-18 13:36:55     at com.epam.ta.reportportal.core.integration.plugin.impl.PluginLoaderImpl.lambda$resolvePluginDetails$3(PluginLoaderImpl.java:103)
2024-11-18 13:36:55     at java.base/java.util.Optional.ifPresent(Optional.java:178)
2024-11-18 13:36:55     at com.epam.ta.reportportal.core.integration.plugin.impl.PluginLoaderImpl.resolvePluginDetails(PluginLoaderImpl.java:101)
2024-11-18 13:36:55     at com.epam.ta.reportportal.plugin.Pf4jPluginManager.uploadPlugin(Pf4jPluginManager.java:344)
2024-11-18 13:36:55     at com.epam.ta.reportportal.plugin.PluginStartUpService.loadLatestVersion(PluginStartUpService.java:64)
2024-11-18 13:36:55     at com.epam.ta.reportportal.plugin.PluginStartUpService.lambda$loadPlugins$0(PluginStartUpService.java:56)
2024-11-18 13:36:55     at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
2024-11-18 13:36:55     at com.epam.ta.reportportal.plugin.PluginStartUpService.loadPlugins(PluginStartUpService.java:56)
2024-11-18 13:36:55     at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
2024-11-18 13:36:55     at java.base/java.lang.reflect.Method.invoke(Method.java:580)
2024-11-18 13:36:55     at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleElement.invoke(InitDestroyAnnotationBeanPostProcessor.java:389)
2024-11-18 13:36:55     at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleMetadata.invokeInitMethods(InitDestroyAnnotationBeanPostProcessor.java:333)
2024-11-18 13:36:55     at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeInitialization(InitDestroyAnnotationBeanPostProcessor.java:157)
2024-11-18 13:36:55     ... 24 common frames omitted
